guides/user/kubernetes-v2/automated-rollbacks/index.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: single
+title:  "Configure Automated Rollbacks in the Kubernetes Provider V2"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.6" %}
+
+{% include toc %}
+
+There are clear benefits to having your stored manifests (in Git, GCS, etc...)
+match as closely as possible to what's running in your cluster: you have
+auditability, versioned changes, and a high confidence in what code you have
+running. However, in some cases, pushing changes from someplace like GitHub
+through your build system all the way production can simply take too long, and
+an escape hatch is needed when something has gone seriously wrong.
+
+When using a Deployment object, you can see the history of rollouts in the
+"Clusters" tab:
+
+{% include
+   figure
+   image_path="./revisions.png"
+   caption="Version 4 is active, and version 3 has no pods running."
+%}
+
+Spinnaker exposes "Undo Rollout" functionality in two places, in the [Clusters
+tab](#ad-hoc-rollbacks), and as a [pipeline stage](#automated-rollbacks).
+
+## Ad-hoc Rollbacks
+
+In cases where you see something is immediately wrong and isolated to a
+resource in the "Clusters" tab, you can select "Undo Rollout" from the
+"Actions" dropdown:
+
+{% include
+   figure
+   image_path="./dropdown.png"
+%}
+
+And select the healthy revision to make active again:
+
+{% include
+   figure
+   image_path="./adhoc.png"
+%}
+
+Notice that the old configuration (version 3) will be rolled forward into a
+new version (version 5):
+
+{% include
+   figure
+   image_path="./v005.png"
+%}
+
+## Automated Rollbacks
+
+You can also configure automated rollbacks inside of Spinnaker pipelines. These
+stages are best configured when other stages or branches fail, indicating that
+something has gone wrong in your rollout.
+
+{% include
+   figure
+   image_path="rollback-stage.png"
+%}
+
+One parameter to watch out for is __Revisions Back__, which counts how many
+revisions the current active revision should be rolled back by. If you have the
+following state:
+
+```
+nginx-deployment-2d8178b77 (Revision 5) # active
+nginx-deployment-7bdd110f7 (Revision 4) 
+nginx-deployment-0b13cc8c1 (Revision 1) 
+```
+
+And roll back by "1" revision, (Revision 3) will be active again. Rollback by
+"2" revisions and (Revision 1) will be active again.
+
+> Keep in mind that Kubernetes will implicitly rollforward the old
+> configuration, creating (Revision 6) in both cases.
```

Comment:
Right, just to make sure I understand, not necessarily that you need to clarify for reader...

Roll back N rolls back to N-1 (for example), but _calls it_ N+1, because now it's a new revision? But it's identical to N-1?

Reply:
Yup that's right, it's a Kubernetes quirk